src/hub.js

```diff
@@ -352,14 +370,31 @@ const onReady = async () => {
   const initialEnvironmentEl = document.createElement("a-entity");
   initialEnvironmentEl.addEventListener("bundleloaded", () => {
     remountUI({ initialEnvironmentLoaded: true });
-    // Wait a tick plus some margin so that the environments actually render.
-    setTimeout(() => scene.renderer.animate(null), 100);
+    // We never want to stop the render loop when were running in "bot" mode.
+    if (!qsTruthy("bot")) {
```

Comment:
i think this should be pulled up to a `const isBotMode` since we do it in a few places